src/client/dfs/dfs.c

```diff
@@ -326,7 +326,7 @@ fetch_entry(daos_handle_t oh, daos_handle_t th, const char *name,
 	if (fetch_sym && S_ISLNK(entry->mode)) {
 		size_t sym_len;
 
-		D_ALLOC(value, PATH_MAX);
+		D_MM_ALLOC(value, PATH_MAX);
```

Comment:
This is used for reading symbolic link targets only, it's not a particularly performance critical area of the code.

Reply:
ok